cms/templates/js/self-paced-due-date-editor.underscore

```diff
@@ -0,0 +1,20 @@
+<ul class="list-fields list-input date-setter">
+    <li class="field field-text field-due-in">
+        <label for="due_in"><%- gettext('Due in:') %></label>
+        <input type="number" id="due_in" name="due_in" value=""
+            placeholder="" autocomplete="off" max="18"/> weeks
+    </li>
+</ul>
+
+<ul class="list-actions">
+    <li class="action-item">
+        <a href="#" data-tooltip="<%- gettext('Clear Due Date') %>" class="clear-date action-button action-clear">
+            <span class="icon fa fa-undo" aria-hidden="true"></span>
+            <span class="sr"><%- gettext('Clear Due Date') %></span>
+        </a>
+    </li>
+</ul>
+
+<div id="warning" class="message-status error">
+    <%- gettext('The maximum number of weeks this subsection can be due in is 18 weeks') %>
```

Comment:
Nit: Period? I'd say go for consistency with other warning messages in the area if there are any.